packages/holocron/src/loadModule.web.js

```diff
@@ -50,16 +50,15 @@ export default function loadModule(moduleName, moduleData) {
     }
     script.src = isProduction ? `${url}?key=${getModuleMap().get('key')}` : url;
     const timeout = setTimeout(onScriptComplete, 120000);
-    script.onerror = (err) => {
-      script.onerror = null;
+    script.addEventListener('error', (event) => {
```

Comment:
was this change from `onerror` to `addEventListener('error'` tested in browsers? ex: IE11

Reply:
Seems to support everything above IE9 https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener